CONTRIBUTING.md

```diff
@@ -254,6 +255,26 @@ Introducing new [Parse Server configuration][config] parameters requires the fol
 1. Add test cases to ensure the correct parameter value validation. Parse Server throws an error at launch if an invalid value is set for any configuration parameter.
 1. Execute `npm run docs` to generate the documentation in the `/out` directory. Take a look at the documentation whether the description and formatting of the newly introduced parameters is satisfactory.
 
+## Versioning
+
+> The following versioning system is applied since Parse Server 5.0.0 and does not necessarily apply to previous releases.
+
+Parse Server follows [semantic versioning](https://semver.org) with a flavor of [calendric versioning](https://calver.org). Semantic versioning makes Parse Server easy to upgrade because breaking changes only occur in major releases. Calendric versioning gives an additional sense of how old a Parse Server release is and allows for future Long-Term Support concepts, which Parse Server currently does not provide.
+
+Syntax: `[major]`**.**`[feature]`**.**`[fix]`**-**`[pre-release-label]`
```

Comment:
Why not stay with the conventions ?

`[feature]` => `[minor]`
`[fix]` => `[patch]`

Reply:
Yes we could do that; I'm thinking ahead about branch names to manage releases. If we name the branches `fix`, `feature`, `breaking`, contributors would intuitively know against which branch to open a PR. Just a though, since the branches are still in discussion.

Reply:
I renamed it to the traditional names as you suggested. Once we decide on the branch structure, we can always come back and rename accordingly.